### DISCUSSION TITLE
Tag Notes in Breadcrumbs V4

### DISCUSSION THREAD

**Discussion by wicket-quest** [SkepticMystic/breadcrumbs#560] [General]
A few days ago, I created a new vault called Questions and Answers to store web clippings and annotations about Q&A forums, including Stack Overflow. So far, I have a few notes, and I'm thinking that Breadrumbs might be helpful. However, I can't get Tag Notes to work, so I installed Breadcrumbs V4 using Brat, but I'm still having problems with this hierarchy. The Dates and Folder Notes hierarchies are working fine.

I have created a very simple note to test this feature
```
---
BC-tag-note-tag: "#javascript"
---
# Exploring Breadcrumbs Tag Note
```

I have three notes with tags property, including javascript tag and other tags. All these notes are in the same folder. The path is `Stack Exchange/stackoverlow`. This folder has a folder note that is working fine.

Below is the info about my environment:

**H&S**
- MacOs Sonoma 14.5
- Obsidian for Mac Version 1.6.5 (Installer 1.5.3)

**Plugins**

- BRAT
- Breadcrumbs 4.2.35-beta
- Dataview 0.5.67
- Enveloppe 7.2.8
- Multi Properties 1.1.1
- Sort and Permute lines
- Templater 2.3.3

I'm unsure if this is because of a bug or if I'm missing something. Please advise.

**Related Issues**

#361 
#401 

**Related Discussions**

I have not found a discussion that is specific about using Tag Note.

**Related Docs**

- https://publish.obsidian.md/breadcrumbs-docs/Explicit+Edge+Builders/Tag+Notes
- https://breadcrumbs-wiki.onrender.com/docs/Alternative%20Hierarchies/Tag%20Notes

**Comment by wicket-quest** [comment]
Re-reading my post and the related docs, I re-tested (I'm sure to have done the same tests yesterday) and found that the "correct" syntax is the following:
```
---
BC-tag-note "#javascript"
---
# Exploring Breadcrumbs Tag Note
```

**Comment by SkepticMystic** [comment]
Hey there, thank you for the detailed issue.

Please can you share an example of a note with the javascript tag. I have a feeling you might be adding the tag in a way I haven't accounted for

**Reply by wicket-quest** [reply]
Thanks for your speedy reply... I found the "problem": I was using `BC-tag-note-tag` instead of `BC-tag-note` (I mentioned this in a comment to my post a few "seconds" ago.

**Reply by wicket-quest** [reply]
@SkepticMystic  As mentioned in a previous comment, I created a new vault. It only has Brat and Breadcrumbs plugins.

Note with tag.md
```md
---
tags:
  - javascript
---
# Note with tag
This note has javascript as a tag.
```
Tag Note.md
```md
---
BC-tag-note-tag: javascript
---
```

I opened the Breadcrumbs Tree view and refreshed the index. Nothing happened. The Obsidian console shows the following

![Captura de pantalla 2024-07-05 a la(s) 10 35 31](https://github.com/SkepticMystic/breadcrumbs/assets/132358235/063df56a-20aa-497a-b50b-0ae8b2ec0ff6)

Using `#javascript` instead of `javascript` makes no difference.

**Reply by wicket-quest** [reply]
I tried changing `BC-tag-note-tag` to `BC-tag-note`. This didn't work either. I think that the Questions and Answers worked because first, I installed the "old" version of Breadcrumbs, and then I installed Bratt and Breadcrumbs beta.

**Reply by wicket-quest** [reply]
The problem happens with frontmatter tags. With inline tags, apparently, it works fine.

**Comment by wicket-quest** [comment]
@SkepticMystic 

Today, Breadcrumds has crashed on my Questions and Answers vault. Also, I learned the shortcut to open the console using a Command + Option + I. About the topic of this thread, I noticed the following entry
```
[BC:WARN][15:14:05.446Z] 
 'BC-tag-note' is deprecated in favor of BC-tag-note-tag
plugin:breadcrumbs:9665
```

There are two other messages, one about using an invalid value and another about a close_field. I will add the details a bit later.

Also, I will try to reproduce the problem depicted on the OP on a new vault and see if the Breadcrumbs crashes on this new vault.
